core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionFixture.scala

```diff
@@ -0,0 +1,82 @@
+package org.broadinstitute.dsde.rawls.expressions
+
+// test data for Expression Validation and Parsing
+
+trait ExpressionFixture {
```

Comment:
this is a fantastic resource to use as a description of "what is an isn't valid". thank you for putting this together!